cherrypy/test/helper.py

```diff
@@ -461,11 +461,10 @@ def start(self, imports=None):
         ```
         ['-c',
          "__requires__ = 'CherryPy'; \
-         import pkg_resources, re, sys; \
+         import importlib.metadata, re, sys; \
```

Comment:
It seems this won't work on Python 3.6/3.7 and needs a fallback.

Reply:
Use the pypi package for the older python versions?
https://pypi.org/project/importlib-metadata/

Reply:
Yes, it's already used in one place in the PR.

Reply:
I'm working on the patch, I'll push up a new pr soon

Reply:
@radez oh wait, no need to change this specific line — it's just a part of the explanation within a docstring.

The only place where this is needed is `docs/conf.py` — it's failing on RTD.

Reply:
ack, got it.